scripts/npm/unlink/index.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+var path = require( 'path' );
+var fs = require( 'fs' );
+var componentsDir = path.join( __dirname, '..', '..', '..', 'components' );
+var components = [ 'cf-buttons',
```

Comment:
@Scotchester figured out how to do it. Ready for final review and then I'll duplicate this on the other repo.